apps/www/app/(dashboard)/dashboard/settings/connectors/connector-cards.tsx

```diff
@@ -0,0 +1,166 @@
+import Image from "next/image";
+
+import BlurImage from "@/lib/blur-image";
+import { Badge } from "@/components/ui/badge";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { Separator } from "@/components/ui/separator";
+import {
+  Sheet,
+  SheetClose,
+  SheetContent,
+  SheetDescription,
+  SheetFooter,
+  SheetHeader,
+  SheetTitle,
+  SheetTrigger,
+} from "@/components/ui/sheet";
+
+// Define the connectors array with example data
+const connectors = [
```

Comment:
I'll provide the APIs for retrieving this in another issue and integrate it. Ok?